lib/dealership.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_support'
+require 'active_support/core_ext/numeric/conversions'
+require 'pry'
+
+
+class Dealership
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def inventory_count
+    @inventory.count
+  end
+
+  def has_inventory?
+    return true if @inventory.count > 0
+    false
+  end
+
+  def cars_by_make(make)
+    @inventory.select { |car| car if car.make == make }
```

Comment:
Nice work with using the `select` enumerable, as well as utilizing the shorthand method throughout this class.